pkgs/top-level/aliases.nix

```diff
@@ -437,10 +437,6 @@ mapAliases ({
     gst-plugins-ugly = pkgs.gst-plugins-ugly;
   };
 
-  # added 2019-08-01
-  mumble_git = pkgs.mumble_rc;
-  murmur_git = pkgs.murmur_rc;
```

Comment:
I think these should stay but point to `pkgs.{mumble,murmur}` instead. This then won't break people's configs.

Reply:
Thank you for your feedback, aliases have been re-added.